### DISCUSSION TITLE
How can I use Filter Plugin with Ant Media Server?

### DISCUSSION THREAD

**Discussion by USAMAWIZARD** [ant-media/Ant-Media-Server#4982] [FAQs] [answered]
I see that we can use Filter Plugin with AMS in order to crate audio-video manipulations on the FLY.
How can I add a logo to the stream by using Filter Plugin!
Also, Can I use Filter Plugin with Community Edition!

**Comment by yashtandon113** [comment, marked as answer]
1. Ant Media Server Filter Plugin lets you make Real-time Audio/Video Manipulation on the server-side as told in this [blogpost](https://antmedia.io/discover-the-filter-plugin/). Filter can be created on the fly by the REST API.

2. You can add logo to an on going stream by applying movie filter to the stream. Here is the Rest API sample:

`curl -i -X POST -H "Accept: Application/json" -H "Content-Type: application/json" "http://localhost:5080/WebRTCAppEE/rest/v2/filters/create" -d '{"filterId":"logodemo","inputStreams":["stream1"],"outputStreams":["test"],"videoFilter":"movie=/home/burak/antmedia.png[logo];[in0][logo]overlay[out0]","audioFilter":"[in0]acopy[out0]","videoEnabled":"true","audioEnabled":"true","type":"asynchronous"}'
`

**Note:** You need to change the path of the logo/image with your file path in video filter.

3. Filter Plugin feature is only available in Enterprise Edition of Ant Media Server.

**Comment by josetowers** [comment]
How to place a logo on the server side, I can't find relevant information
